src/utils/event.js

```diff
@@ -0,0 +1,47 @@
+import dayjs from 'dayjs';
+import duration from 'dayjs/plugin/duration';
+
+const DatetimeFormat = {
+  MINUTE: 'mm[M]',
+  HOUR: 'HH[H] mm[M]',
+  DAY: 'DD[D] HH[H] mm[M]',
+};
+
+const DatetimeLimit = {
+  MINUTE: 60,
+  HOUR: 24,
+};
+
+export const formatDate = (date, format) => {
+  return dayjs(date).format(format);
+};
+
+export const getTimeDuration = (fromDate, toDate) => {
+  dayjs.extend(duration);
+
+  const startDate = dayjs(fromDate);
+  const endDate = dayjs(toDate);
+
+  const difference = endDate.diff(startDate);
+  const timeDuration = dayjs.duration(difference);
+
+  if (endDate.diff(startDate, 'minute') < DatetimeLimit.MINUTE) {
+    return timeDuration.format(DatetimeFormat.MINUTE);
+  } else if (endDate.diff(startDate, 'hour') < DatetimeLimit.HOUR) {
+    return timeDuration.format(DatetimeFormat.HOUR);
+  } else {
+    return timeDuration.format(DatetimeFormat.DAY);
+  }
```

Comment:
а давай просто лесенку if сделаем, а в конце без else?
```suggestion
  if (endDate.diff(startDate, 'minute') < DatetimeLimit.MINUTE) {
    return timeDuration.format(DatetimeFormat.MINUTE);
  }
  
   if (endDate.diff(startDate, 'hour') < DatetimeLimit.HOUR) {
    return timeDuration.format(DatetimeFormat.HOUR);
  }
  
  return timeDuration.format(DatetimeFormat.DAY);
```